### PR TITLE
Add kontakt text

### PR DESCRIPTION
**Pull request by CyrilleKolle**
Link to ticket: https://fixa.itiden.se/project/1362/tickets/90

### CODE REVIEW

**Review by ricllrstn — CHANGES_REQUESTED** [review]
Textblock is offset to the left. Should be same as "hem".

**Review by ricllrstn — COMMENTED** [review]
Nice work!

**Review by ricllrstn — APPROVED** [review]

File: pages/kontakt.tsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ const ContentWrapper = styled.div`
   ${tw`max-w-screen-xl px-6 mx-auto items-center md:(mt-0 px-5) xl:(max-w-screen-lg px-0 mx-auto)`}
 `;
 const QuarteredContent = styled.div`
-  ${tw`flex flex-col pt-56 md:(w-3/4 pt-0)`}
+  ${tw`flex flex-col pt-56 md:( pt-0)`}
```
Comment (reekris) [inline pages/kontakt.tsx:31]: remove space in `md:( pt-0)`

File: pages/kontakt.tsx [hunk 2]
```diff
@@ -40,6 +41,7 @@ const QuarteredContent = styled.div`
 const GrayBanner = styled(Hero)`
   ${tw`mt-16 py-10 bg-gray-700`}
 `;
+
```
Comment (reekris) [inline pages/kontakt.tsx:44]: Remove extra line break

File: pages/kontakt.tsx [hunk 3]
```diff
@@ -58,11 +60,22 @@ const Adresswide = styled.a`
 const Divider = styled.div`
   ${tw`w-full h-px bg-gray-600 mb-8 last:hidden md:hidden`};
 `;
+const ContactLinkTop = styled.a`
```
Comment (reekris) [inline pages/kontakt.tsx:63]: Add line breaks between components

**Review by reekris — CHANGES_REQUESTED** [review]
Some minor changes. Also there is a typo in the PR description :)

**Review by reekris — CHANGES_REQUESTED** [review]
- Still some `margin={true}` that can be removed. Check "Files changed" here on github 
- Also fix typo in PR title.

**Comment by CyrilleKolle** [comment]
I realised "560px-40px" is not the same as 520px so the changes I made that were subsequently merged in to the master didn't show the desired Hero.

**Review by ricllrstn — APPROVED** [review]

**Review by reekris — CHANGES_REQUESTED** [review]
Make HeaderWide and its children into a reusable component.

- reuse this component on index and kontakt pages

File: components/Layout/HeroContent.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```
Comment (reekris) [inline components/Layout/HeroContent.tsx:1]: Could we rename this component to `Hero`? I Think its more concise than `HeroContent`

File: pages/kontakt.tsx [hunk 5]
```diff
@@ -3,12 +3,13 @@ import React from 'react';
...
 import tw from 'twin.macro';
 import { IndexHeader } from '../components/IndexHeader';
-import { Page, Hero, H3 } from '../components/Layout';
+import { Page, Hero, H3} from '../components/Layout';
 import { ProfileCard } from '../components/ProfileCard';
 import { Employee } from '../models/Employee';
 import { SocialmediaGroupLarge } from '../components/SocialMediaIcons/SocialmediaGroupLarge';
 import { Map } from '../components/Map/Map';
 import { getEmployeesFromJson } from '../data/employee';
+import {HeroContent} from '../components/Layout/HeroContent'
```
Comment (reekris) [inline pages/kontakt.tsx:12]: Missing spaces between brackets

File: pages/kontakt.tsx [hunk 6]
```diff
@@ -17,28 +18,11 @@ interface IndexPageProps {
...
-    min-width: 620px;
-  }
-
-  @media screen and (max-height: 580px) {
-    ${tw`pt-40 pb-32`}
-  }
-`;
 const GrayBanner = styled(Hero)`
-  ${tw`mt-16 py-10 bg-gray-700`}
+  ${tw` py-10 bg-gray-700`}
```
Comment (reekris) [inline pages/kontakt.tsx:25]: Remove whitespace in beginning of string literal

**Review by reekris — CHANGES_REQUESTED** [review]
Nice! We are really close, just some super minor stuff

**Review by ricllrstn — CHANGES_REQUESTED** [review]
Some minor tweeks:
– Media query for min-height can be set to 610px instead of 780px
– For me it shows ~80px instead of 40px of the next section (Same for "Hem")

**Review by ricllrstn — DISMISSED** [review]
Great stuff!

File: components/Layout/HeaderWide.tsx [hunk 7]
```diff
@@ -2,9 +2,10 @@ import React from 'react';
 import styled from 'styled-components';
 import tw from 'twin.macro';
 
-export const Wrapper = styled.div<{ viewportHeight: string }>`
+export const Wrapper = styled.div`
   ${tw`flex flex-col justify-end -mt-10`}
-  height: ${p => p.viewportHeight};
+  min-height: 440px;
+  height: 100vh;
```
Comment (filipengberg) [inline components/Layout/HeaderWide.tsx:8]: Use tailwind classes instead of hardcoded values

File: components/Layout/HeroContent.tsx [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+import { H1, HeaderWide, Content, Arrow } from '.';
+import { Tags } from '../Tag';
+
+type HeroProps = {
+  tags: string[];
+  showArrow?: boolean;
+};
+
+const TagsWrapper = styled.div`
+  ${tw`w-full mt-1 mb-12 hidden sm:block `};
```
Comment (filipengberg) [inline components/Layout/HeroContent.tsx:13]: Remove white space at the end

File: pages/kontakt.tsx [hunk 9]
```diff
@@ -17,34 +18,17 @@ interface IndexPageProps {
...
-  ${tw`flex flex-col pt-56 md:(w-3/4 pt-0)`}
-
-  @media screen and (min-width: 768px) {
-    min-width: 620px;
-  }
-
-  @media screen and (max-height: 580px) {
-    ${tw`pt-40 pb-32`}
-  }
+  ${tw`max-w-screen-xl px-6 mx-auto items-center mb-32 md:(mt-0 px-5 mb-0) xl:(max-w-screen-lg px-0 mx-auto)`}
```
Comment (filipengberg) [inline pages/kontakt.tsx:22]: mx-auto not needed in xl since it is default

**Review by filipengberg — CHANGES_REQUESTED** [review]
Some minor stuff

**Review by filipengberg — APPROVED** [review]

**Pull request merged**